internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/DaVinciClientDiskFullTest.java

```diff
@@ -207,6 +208,8 @@ public void testDaVinciDiskFullFailure(boolean useDaVinciSpecificExecutionStatus
     String inputDirPath = "file://" + inputDir.getAbsolutePath();
     Schema recordSchema = TestWriteUtils.writeSimpleAvroFileWithStringToStringSchema(inputDir, 1, 10);
     Properties vpjProperties = defaultVPJProps(venice, inputDirPath, storeName);
+    vpjProperties.putAll(
+        PubSubBrokerWrapper.getBrokerDetailsForClients(Collections.singletonList(venice.getPubSubBrokerWrapper())));
```

Comment:
Should it be merged into defaultVPJProps? Also I am a bit confused could you explain which part is using this?

Reply:
Yeah, we’re planning to replace it eventually. But for now, I’m skipping it because doing so
would require updating all the tests, which I’d prefer to defer.

> Also I am a bit confused could you explain which part is using this?


There are some PubSub-specific properties that need to be passed down to the PubSub adapters.
Depending on the type of PubSub backend, these properties can vary. Each PubSub backend
needs to override the client properties API to supply the appropriate custom properties,
which ensures the corresponding clients are created correctly.